README.md

```diff
@@ -297,55 +298,68 @@ and it is far easier to work with them when you have deeper understanding yourse
   trade. In the United States,
   [several kinds of business entities](https://en.wikipedia.org/wiki/Types_of_business_entity#United_States)
-  may be formed, or [**incorporated**](https://en.wikipedia.org/wiki/Incorporation_(business)),
+  may be formed, or [**incorporated**](<https://en.wikipedia.org/wiki/Incorporation_(business)>,
   under the laws of a specific state.
-- 🄳 A [**startup**](https://en.wikipedia.org/wiki/Startup_company) is an emerging company that
-  aspires to grow quickly.
+- 🄳 A [**startup**](https://en.wikipedia.org/wiki/Startup_company) is an emerging company,
+  typically a [private company](#public-and-private-companies), that aspires to grow quickly in
+  size, revenue, and influence.
   Once a company is established in the market and successful for a while, it usually stops
   being called a startup.
-  - ☝️ The term *startup* is pretty informal.
-    It’s not a legal concept, and not everyone uses it consistently.
-    For our purposes, it’s best just to think of it as the young period of a fast-growing
-    company.
+  - ☝️ Unlike some of the legal terms we discuss, the term *startup* is informal, and not
+    everyone uses it consistently.
+    As distinguished investor [Paul Graham]\([https://en.wikipedia.org/wiki/Paul*Graham*(programmer](<https://en.wikipedia.org/wiki/Paul_Graham_(programmer>))
+    [emphasizes](http://www.paulgraham.com/growth.html)—and certainly this is true for our
+    purposes here—it’s best to think of a startup as any [early stage](#stages-of-a-startup)
+    company intending to grow quickly.
   - 🚧 Groundwork:
     What are good stats on how many people work in startups vs established companies?
 - Companies in the US include sole proprietorships, partnerships, limited liability
   companies (LLCs), S corporations, and C corporations.
   - 🔸 This Guide does not cover equity compensation in LLCs, as they are rarely used as the
-    choice of entity for technology startups, and equity compensation in LLCs is dramatically
-    different from equity compensation in corporations.
-    So we’ll focus on the one type of company most readers of this Guide will be concerned
-    with: The C corporation.
+    choice of entity for startups in technology and in non-tech sectors, and equity
+    compensation in LLCs is dramatically different from equity compensation in corporations.
+    The equity components of sole proprietorships, partnerships, and S corps are equally
+    divergent from equity compensation in typical public and private companies, including
+    startups, which are all overwhelmingly C corporations.
 - 🄳 A [**C corporation (or C corp)**](https://en.wikipedia.org/wiki/C_corporation) is a type of
   business entity in the United States.
-  Specifically, it’s a kind of
-  [**stock corporation**](https://en.wikipedia.org/wiki/Joint-stock_company) with certain federal
+  Specifically, it’s a kind of stock corporation (also called a
```

Comment:
This should somehow be merged with the new definition above, as they're essentially alternate terms. Having "joint-stock company" be boldfaced here like a definition here, but not listed as an alternate above is confusing.

I'd figured we would define stock corporation and C corporation at once, but we could put both right here as separate definitions.